release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -811,6 +857,21 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|NFS support for Azure File CSI Operator Driver
+|Not Available
+|Not Available
+|Generally Available
+
+|Read Write Only Pod access mode
```

Comment:
Can we please update "Read Write Only Pod" to "Read Write Once Pod" ?

Reply:
Done.